manager/integration/tests/common.py

```diff
@@ -29,6 +29,7 @@
 EXPAND_SIZE = str(32 * Mi)
 VOLUME_NAME = "longhorn-testvol"
 DEV_PATH = "/dev/longhorn/"
+TEMP_FILE_PATH = "/tmp/temp_file.cfg"
```

Comment:
1. This tmp file is used in `backupstore_create_file` only, right? I think you can put this definition in the file `backupstore.py`
2. The function `backupstore_create_file` itself is a common function and it's not only for creating invalid backup config files, hence we can remove the surffix `cfg` from the tmp file name used in the function.

Reply:
Great point! Fixed. Thank you